dev-client/src/config/index.ts

```diff
@@ -81,5 +81,4 @@ export const APP_CONFIG = {
   googleClientId,
   googleRedirectURI,
   microsoftRedirectURI,
-  FF_offline: willFlagBeEnabledOnReload('FF_offline'),
```

Comment:
note: i moved this to `featureFlags.ts` because:
- we already expose a function which reads off these `APP_CONFIG` values and it seems better to not have more than one way to do that
- it makes it easier to test `featureFlags.ts` as an isolated module

Reply:
Yay, I like it!